core/src/main/scala/scalacache/serdes/BaseCodecs.scala

```diff
@@ -0,0 +1,112 @@
+package scalacache.serdes
+
+trait BaseCodecs {
+
```

Comment:
Where did this serialisation code come from? If it's lifted from somewhere (Shade?) then we should at least give credit in a code comment.